woocommerce/woocommerce
# Add helper tool for making sure build, lint, and test are run through Turborepo

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds a helper tool for verifying that `build`, `lint`, and `test` scripts cannot be directly run in any of the plugins.

Note: Because of the `prepare` scripts for the `e2e-*` packages, which attempt to build, I wasn't able to apply this treatment to them.



Closes #33459.

### How to test the changes in this Pull Request:

1. `pnpm install`
2. Attempt to build a project directly with `pnpm`, such as: `cd plugins/woocommerce && pnpm run build`
3. Verify that you're given an error message and that the error message gives you sensible instructions for actually building the project with Turborepo.
4. Repeat the previous steps with `lint` and `test` for any projects that have those scripts defined.

### Other information:




### FOR PR REVIEWER ONLY:
